src/js/Controllers/DisplayCapabilities.js

```diff
@@ -421,7 +421,7 @@ let capabilities = {
 			"displayType": "SDL_GENERIC",
 			"displayName": "GENERIC_DISPLAY",
 			"textFields": [
-				textField("scrollableMessageBody", 44, 15),
+				textField("scrollableMessageBody", 44, 8),
```

Comment:
While we are fixing these, I noticed that there are some `textField` entries in the `imageFields` array in this template, might be worth removing those in this PR as well